tests/requirements-libraries.txt

```diff
@@ -17,8 +17,6 @@
 #
 # - v. 2.2 and above fails.
 Django==2.1.8  # pyup: ignore
-matplotlib==3.0.3; python_version <= '3.5'  # pyup: ignore
-matplotlib==3.2.2; python_version >= '3.6'  # pyup: ignore
```

Comment:
This will complety remove testing matplot. How is this related to Python 3.9?

Reply:
matplot (and pandy and scipy) requires numpy. But Numpy is not yet ready for Python 3.9.

BUT I just saw Numpy 1.19.3 is out and it works on Python 3.9. I will update everything.